### ISSUE TITLE
Could you please provide the PSNR/SSIM of 8X SR?

### ISSUE THREAD

**Issue by LuyueBUPT** [Zheng222/IDN-Caffe#12]
I recently use this code to train a 8X SR IDN model.  Unfortunately, I get very poor results, even lower than VDSR. So, I wonder if you could provide the PSNR/SSIM of 8X SR? Since I want to cite your work in my paper.

**Comment by LuyueBUPT** [comment]
We use X2 caffemodel provided by this work to initialize the 8X SR. The input size is 9/17, so the label size is 65/129. I did not change the hyper-parameter of solver.

I notice that this work may be for the fine-tuning. IDN paper do not contain much description about the differences of configuration between training and fine-tuning ,except loss function (L1 L2), the number of iterations (100000 vs 600000 ?).

**Comment by Zheng222** [comment]
@LuyueBUPT Hello, I didn't train the ×8 model, so I can't provide you with the results. If you want to compare with the state-of-the-art methods, I recommend that you chose [DBPN](https://github.com/alterzero/DBPN-Pytorch) and [RCAN](https://github.com/yulunzhang/RCAN), which have the  ×8 models.

**Comment by LuyueBUPT** [comment]
Thanks for your help.
The two methods are trained with DIV2K dataset, while ours with 290 datasets. Your method is the most recent method with 290 datasets. 

Could you please tell me the key point of training a ×8 IDN? I use L1 loss and 100,000 iters in training stage which are mentioned in paper, and use L2 loss and 600,000 iters in fine-tuning stage which are provided by solver.prototxt. But the whole work only takes 7 hours with a 1080ti. I wonder if I got something wrong.
Big thanks!

**Comment by Zheng222** [comment]
@LuyueBUPT I guess that you could misunderstand our training strategy. We pre-train our model 100,000 iterations due to the problem of PyCaffe (I implement upsample layer by python layer in the original version). However, I release the code in this repository can directly train  600,000 iterations and then fine-tune 100,000 iterations.

**Comment by LuyueBUPT** [comment]
@Zheng222 OK, I will try this later. Thanks for your assistance.
